### PR TITLE
Syndicate (blood-red) magboots

### PR DESCRIPTION
**Pull request by kol1th**
image
image
offstate + on state

straight outta PARADISE

### CODE REVIEW

**Comment by jakmak** [comment]
does this remove all the normal yellow mag boots from the nuke op base? If so be sure to add some normal mags near the 2 red space suits for stealth ops

**Comment by kol1th** [comment]
since it'd be mapping it'd be a different PR

i'll consider it

File: code/modules/clothing/shoes/magboots.dm [hunk 1]
```diff
@@ -97,4 +97,12 @@
...
\ No newline at end of file
+	mag_slow = 0
+
+//Syndicate
+/obj/item/clothing/shoes/magboots/syndie
+	name = "blood-red magboots"
+	desc = "Reverse-engineered magnetic boots that have a heavy magnetic pull. Property of Gorlex Marauders."
+	icon_state = "syndiemag0"
+	base_state = "syndiemag"
+	species_fit = list(VOX_SHAPED)
```
Comment (ComicIronic) [inline code/modules/clothing/shoes/magboots.dm:108]: Why do this? Vox have their magclaws, and there'll never be a newcops Vox.

**Comment by TheFriggenSawbones** [comment]
COMIC
THERE ARE NUKEOPS VOX

File: code/modules/clothing/shoes/magboots.dm (hunk 1 above)
Comment (JustSumBody) [reply code/modules/clothing/shoes/magboots.dm:108]: There are definitely Nuke Op Vox. Have their own hardsuits and everything.

**Comment by ComicIronic** [comment]
Why the hell are there nukeop Vox
Who is responsible for this abomination

**Comment by kol1th** [comment]
@ComicIronic There are nuke ops for EVERY race. Greys, tajara, unathi, vox, skrell, humans.

**Comment by TheFriggenSawbones** [comment]
comic i'm not trying to be mean here but who are you do you play the game i've always wondered this

are you just some guy who happened upon our codebase and decided to help out

File: code/modules/clothing/shoes/magboots.dm (hunk 1 above)
Comment (TheFriggenSawbones) [reply code/modules/clothing/shoes/magboots.dm:108]: hi

**Comment by Durgen** [comment]
new cops vox are best new cops

**Comment by Kurfursten** [comment]
> Merging without testing
> Invisible magboots

JUST

**Comment by JustSumBody** [comment]
Wasn't this PR that broke it, it was his later one with the workboots. He rebased wrong and used the out of date .dmi.

**Comment by Kurfursten** [comment]
Wew, that means I don't need to apologize.

**Pull request merged**